### DISCUSSION TITLE
Logging in issues

### DISCUSSION THREAD

**Discussion by zhtgeneral** [vercel/vercel#10448] [Help]
### Question

How do I log in if:
I registered with a github account,
logging in with github account gives error (user is blocked + no other info),
registering with email normally gives error (account with email is already created, probably referring to the previous github account),
trying to log in with that email gives error (no contact given, so it looks like Vercel thinks I'm malicious)

No verification or anything was sent to my email

**Comment by alvinsjoy** [comment]
You will have to contact [support@vercel.com](mailto:support@vercel.com). I too had the same problem: #10000
